src/main/java/com/flexrate/flexrate_back/member/application/SignupMemberService.java

```diff
@@ -0,0 +1,69 @@
+package com.flexrate.flexrate_back.member.application;
+
+import com.flexrate.flexrate_back.common.exception.ErrorCode;
+import com.flexrate.flexrate_back.common.exception.FlexrateException;
+import com.flexrate.flexrate_back.member.domain.Member;
+import com.flexrate.flexrate_back.member.domain.repository.MemberRepository;
+import com.flexrate.flexrate_back.member.dto.SignupRequestDTO;
+import com.flexrate.flexrate_back.member.enums.MemberStatus;
+import com.flexrate.flexrate_back.member.enums.Sex;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SignupMemberService {
+
+    private final MemberRepository memberRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    @Autowired
+    public SignupMemberService(MemberRepository memberRepository,
+                               PasswordEncoder passwordEncoder) {
+        this.memberRepository = memberRepository;
+        this.passwordEncoder = passwordEncoder;
+
+    }
+
+    /*
+     * 회원 가입 처리
+     * @param signupDTO 회원가입 요청 데이터
+     * @return 저장된 회원 객체
+     * @throws FlexrateException 이메일 누락, 형식 오류, 중복 또는 비밀번호 유효성 검사 실패 시 예외 발생
+     * @since 2025.04.29
+     * @author 윤영찬
+     */
+    public Member registerMember(SignupRequestDTO signupDTO) {
```

Comment:
controller에서 service 호출 후 즉시 DTO를 반환해 줄 수 있도록 service 로직에서 DTO 변환을 한 뒤 해당 타입으로 반환해주도록 바꿔주세요!

```suggestion
    public SignupResponseDTO registerMember(SignupRequestDTO signupDTO) {
```